datasetinsights/io/downloader/http_downloader.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import os
+import tempfile
+import zipfile
+
+from datasetinsights.io.download import download_file, validate_checksum
+from datasetinsights.io.downloader.base import DatasetDownloader
+from datasetinsights.io.exceptions import ChecksumError
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPDownloader(DatasetDownloader, protocol="http://"):
+    """ This class is used to download data from any HTTP or HTTPS public url
+        and perform function such as downloading the dataset and checksum
+        validation.
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
Not required to have __init__ method.

Reply:
This will fail if the constructor is called with an argument like access_token.

Reply:
`__init__` method should lookup it's parent class `DatasetDownloader` which should be able to catch **kwargs?